user_guide/feature_contexts/hooking_into_the_test_process.rst

```diff
@@ -160,33 +162,39 @@ There are two suite hook types available:
 * ``@BeforeSuite`` - executed before any feature runs.
 * ``@AfterSuite`` - executed after all features have run.
 
-Feature Hooks
--------------
+Tagged Hooks
```

Comment:
Yes & No. Yes, when you are saying that this part documents how one can filter using hooks. And no because even if the hook is helping Behat to filter out what to test, it's still a hook. 

Currently we only have two sections in the documentation about filtering. One is located in the chapter 'Command Line Tool' and the other in the chapter 'Configuration'. I think moving this section to one of these chapters would even make no sense at all.